web/client/components/data/featuregrid/editors/NumberEditor.jsx

```diff
@@ -0,0 +1,61 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const { editors } = require('react-data-grid');
+const nanToNull = v => isNaN(v) ? null : v;
+const processValue = (obj, func) => Object.keys(obj).reduce((acc, curr) => ({
+    ...acc,
+    [curr]: nanToNull(func(obj[curr]))}),
+{});
+const parsers = {
+    "int": v => parseInt(v, 10),
+    "number": v => parseFloat(v, 10)
+};
+class NumberEditor extends editors.EditorBase {
+    static propTypes = {
+      value: PropTypes.oneOfType([
+          PropTypes.string,
+          PropTypes.number]),
+      onBlur: PropTypes.func,
+      inputProps: PropTypes.object,
+      dataType: PropTypes.string,
+      isValid: PropTypes.func,
+      column: PropTypes.object
+    };
+    static defaultProps = {
+        isValid: () => true,
+        dataType: "number"
+    };
+    constructor(props) {
+        super(props);
+        this.validate = (value) => {
```

Comment:
It was my first attempt, but it didn't worked. Probably because it is a derived class.